components/Layout.tsx

```diff
@@ -42,10 +42,10 @@ export default function Layout({ nav, toc, children }) {
         )}
       >
         <Link href="/">
-          <a>
+          <a aria-label="Poimandres Docs">
             <div className="h-full flex items-center flex-none p-2 pl-4 sm:pl-6 xl:pl-8 lg:w-60 xl:w-72">
```

Comment:
drcmda touched upon this in https://github.com/pmndrs/website/commit/0f03414a787b9728becf7c2ad17b08118f2f3066, so I took the liberty to touch up on some accessibility issues.